### ISSUE TITLE
Support for Kong 0.10rc1

### ISSUE THREAD

**Issue by pouledodue** [PGBI/kong-dashboard#70]
Kong 0.10rc1 has new API routing features

https://github.com/Mashape/kong/pull/1970

https://github.com/Mashape/kong/releases/tag/0.10.0rc1

**Comment by Platzii** [comment]
0.10.0rc3 has been released several days ago

https://github.com/Mashape/kong/releases/tag/0.10.0rc3

**Comment by shaowin16** [comment]
Yes,i'm expectant for support kong: 0.10.0

**Comment by mbugeia** [comment]
Hi, any news on this ?

**Comment by maxtuzz** [comment]
0.10 has now been officially released.

**Comment by PGBI** [comment]
So, I'm aware of the last kong release. Didn't have time to test it yet. Does anybody have an exhaustive list of things to be fixed in Kong-dashboard to be compatible with Kong 0.10?

Also, feel free to submit PRs :)

**Comment by jhenry82** [comment]
The most significant change (in terms of kong-dashboard being pretty broken without it) the expansion and renaming of the fields describing an API. Compare https://getkong.org/docs/0.10.x/admin-api/#add-api vs https://getkong.org/docs/0.9.x/admin-api/#add-api.

Adding an API in 0.9.x:
```
$ curl -i -X POST \
  --url http://localhost:8001/apis/ \
  -d 'name=mockbin' \
  -d 'upstream_url=http://mockbin.com/' \
  -d 'request_host=mockbin.com' \
  -d 'request_path=/status'
```
Adding an API in 0.10.x:
```
$ curl -i -X POST http://localhost:8001/apis/ \
    -d 'name=my-api' \
    -d 'upstream_url=http://my-api.com' \
    -d 'hosts=example.com,service.com' \
    -d 'uris=/my-api' \
    -d 'methods=GET,HEAD'
HTTP/1.1 201 Created
```

Instead of request_host and request_path, you now pass hosts (which can be a comma-separated list of allowable hosts) and uris (again as a list). The API returns these as JSON arrays
```
hosts: ["example.com" , "service.com"]
```

There are also a bunch of new optional parameters for routing by method (GET,POST,etc), determining HTTPS behavior, and specifying various timeouts.

**Comment by Boggin** [comment]
Is this linked to #85 ?

**Comment by haha1903** [comment]
Please check PR #86 @PGBI

**Comment by EvanDylan** [comment]
@haha1903 thx for your work

**Comment by PGBI** [comment]
Kong-dashboard 2.0.0 has been released and is compatible with Kong 0.10.x
